suites/quincy/cephadm/tier-1_service_apply_spec.yaml

```diff
@@ -0,0 +1,241 @@
+#       - registry-json: <registry-URL>
+#       - fsid: <cluster-fsid>
+#       - mon-ip: <monitor IP address: Required>
+#   (2) Copy SSH keys to nodes.
+#   (3) Add nodes to cluster with address and role labels attached to it using Host spec yaml file.
+#   (4) Deploy services using apply spec option, (" ceph orch apply -i <spec_file>)
+#       - 3 Mon on node1, node2, node3 using host placements.
+#       - MGR using placement using label(mgr).
+#       - addition of OSD's using "all-avialable-devices" option.
+#       - alertmanager on node1, node2 using label "alert-manager".
+#       - grafana and prometheus on node1 using host placement with limit.
+#       - crash and node-exporter on all nodes using placement="*".
+#   (5) Deploy MDS service using spec file,
+#       (a) Create FS volume 'cephfs'
+#       (b) Deploy MDS with label(mds) option using 'cephfs' volume.
+#   (6) Deploy NFS service using spec file,
+#       (a) Create OSD pool 'nfs-rgw-pool'
+#       (b) Deploy NFS on node4 using 'nfs-rgw-pool' pool.
+#===============================================================================================
+tests:
+  - test:
+      name: Install ceph pre-requisites
+      desc: installation of ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: Cephadm Bootstrap
+      desc: cephadm cluster bootstrap
+      module: test_bootstrap.py
+      polarion-id: CEPH-83573720
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          skip-monitoring-stack: true
+          orphan-initial-daemons: true
+          registry-json: registry.redhat.io
+          custom_image: true
+          mon-ip: node1
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Host addition with spec file
+      desc: add hosts using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574726
+      config:
+        steps:
+        - config:
+            service: host
+            command: set_address
+            args:
+              node: node1
+        - config:
+            service: host
+            command: label_add
+            args:
+              node: node1
+              labels: apply-all-labels
+        - config:
+            command: apply_spec
+            service: orch
+            specs:
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node2
+                 - node3
+             - service_type: host
+               address: true
+               labels: apply-all-labels
+               nodes:
+                 - node4
+      abort-on-fail: true
+  - test:
+      name: Service deployment with spec
+      desc: Add services using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83573746,CEPH-83574727
```

Comment:
Could we revisit having multiple Polarion IDs?

Reply:
Thanks @psathyan
- Polarion IDs revisit completed.
- For initial "quincy" Tier 1 test run we have added same "pacific" Tier1 after testing with ceph6.0 build
https://github.com/red-hat-storage/cephci/blob/master/suites/pacific/cephadm/tier-1_service_apply_spec.yaml
- We are planning to add more Tier 1 suite in upcoming release with more test coverage after discussion.

Reply:
Yes, I understand that it was part of `pacific`... Since the recommendation is to have one polarion per test case... we are enforcing the check now.

Reply:
Hi @psathyan 
As mentioned above i have created two test cases with one polarion test case each.

Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UK8WUB

Reply:
Thanks for fixing it. The changes look good to me.